cita-executor/core/src/libexecutor/fsm.rs

```diff
@@ -139,311 +139,305 @@ impl FSM for Executor {
     }
 }
 
-#[cfg(test)]
```

Comment:
why delete this test case? I thought I have fix all this.

Reply:
Noop
```sh
wpf@wpf:~/working/github/ouwenkg/cita/cita-executor$ cargo test -p core-executor
   Compiling core-executor v0.1.0 (/home/wpf/working/github/ouwenkg/cita/cita-executor/core)
error[E0599]: no method named `commit` found for type `std::sync::Arc<std::cell::RefCell<cita_vm::state::state::State<trie_db::TrieDB<cita_db::RocksDB>>>>` in the current scope
   --> cita-executor/core/src/libexecutor/fsm.rs:187:38
    |
187 |                 executed_block.state.commit().expect("commit state");
    |                                      ^^^^^^

error[E0599]: no method named `commit` found for type `std::sync::Arc<std::cell::RefCell<cita_vm::state::state::State<trie_db::TrieDB<cita_db::RocksDB>>>>` in the current scope
   --> cita-executor/core/src/libexecutor/fsm.rs:191:38
    |
191 |                 executed_block.state.commit().expect("commit state");
    |                                      ^^^^^^

error[E0599]: no method named `commit` found for type `std::sync::Arc<std::cell::RefCell<cita_vm::state::state::State<trie_db::TrieDB<cita_db::RocksDB>>>>` in the current scope
   --> cita-executor/core/src/libexecutor/fsm.rs:195:38
    |
195 |                 executed_block.state.commit().expect("commit state");
    |                                      ^^^^^^

error[E0061]: this function takes 2 parameters but 3 parameters were supplied
   --> cita-executor/core/src/libexecutor/fsm.rs:345:24
    |
345 |         executed_block.apply_transaction(
    |                        ^^^^^^^^^^^^^^^^^ expected 2 parameters
```

Reply:
It seems that you hasn't rebase the right code!!
see:
https://github.com/cryptape/cita/blob/remove-gpl/cita-executor/core/src/libexecutor/fsm.rs#L187